rebuild/src/addon.cc

```diff
@@ -0,0 +1,90 @@
+#include "addon.h"
+
+bool is_zero_bytes(
+    const uint8_t *data, const size_t start_byte, const size_t byte_length) {
+    for (size_t i = start_byte; i < byte_length; i++) {
+        if (data[i] != 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+bool is_valid_length(
```

Comment:
It might be worth documenting all of those methods and adding unit tests (e.g., with `GoogleTest`) to be more explicit about the expected behaviour.

Reply:
I test them via JS unit tests with args passed from the JS side but seems like a good idea to add some C level tests as well

Reply:
Just going to stick with the JS tests.  I tested for these pretty thoroughly so I'm confident they are good to go....